website/markdown/docs/plugins/using-plugins.mdx

```diff
@@ -0,0 +1,58 @@
+---
+name: Using plugins
+excerpt: Learn how to extend Tuist with plugins.
+---
+
+# Using Plugins
+
+## Plugins
+
+With plugins you can extend Tuist to add support for custom project description helpers, custom templates, and much more!
```

Comment:
Right now only project description helpers are supported, right?

Reply:
Yes, only helpers right now. Will remove this line here!